server/views/pages/activities/create-an-activity/education-level.njk

```diff
@@ -5,7 +5,9 @@
 
 {% set pageTitle = applicationName + " - Create an activity - education level" %}
 {% set pageId = 'create-activity-education-level-page' %}
-{% set backLinkHref = "qualification" %}
+
+{% set backLinkHref = "/activities/schedule/activities/" + session.createJourney.activityId if session.req.query.fromEditActivity else "qualification" %}
+
```

Comment:
Should the session.req.query.fromEditActivity part be prefixed session? Or just req.query.fromEditActivity?